_includes/_favicon.html

```diff
@@ -4,27 +4,27 @@
 *   
 {% endcomment %}
 {% comment %} Regular Old-Skool Favicon: {% endcomment %}
-	{% if site.favicon-32x32 %}<link rel="icon" sizes="32x32" href="{{ site.url }}/assets/img/{{ site.favicon-32x32 }}">{% endif %}
+	{% if site.favicon-32x32 %}<link rel="icon" sizes="32x32" href="assets/img/{{ site.favicon-32x32 }}">{% endif %}
```

Comment:
This was taking the logo images from the website so couldn't test without this change. I think relative links are better to make the site portable. I don't see any disadvantage of using relative links.